.vscode/launch.json

```diff
@@ -159,80 +153,76 @@
       "internalConsoleOptions": "openOnSessionStart"
     },
     {
-      "name": "Launch Lightning Tests",
```

Comment:
Removed config. Launch Tests - Lightning - VSCode Integration Tests uses the same config.